daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/EngineConfig.scala

```diff
@@ -30,6 +31,26 @@ final case class EngineConfig(
       TransactionVersions.assignValueVersion(allowedOutputTransactionVersions.max),
     )
 
+  private[lf] def toCompilerConfig: Compiler.Config =
```

Comment:
I assume the engine config contains more things than the compiler config. If so, I would find it slightly confusing to call this method `to...` since we're not really converting something isomorphic into a compiler config but rather extract the compiler config.
```suggestion
  private[lf] def getCompilerConfig: Compiler.Config =
```

Reply:
Good point.